src/main.rs

```diff
@@ -3,9 +3,11 @@ use k8s_openapi::apiextensions_apiserver::pkg::apis::apiextensions::v1::{
     CustomResourceDefinition, CustomResourceDefinitionVersion,
 };
 use kopium::{analyze, OutputStruct};
-use kube::{Api, Client, ResourceExt};
+use kube::{api, Client, ResourceExt};
 use quote::format_ident;
-use structopt::StructOpt;
+use structopt::{clap, StructOpt};
+
+type Crds = api::Api<CustomResourceDefinition>;
```

Comment:
This type is a bit confusing to me imo. I would prefer importing `CustomResourceDefinition as CRD` and using `Api<CRD>` everywhere.

Reply:
No problem